WireUI/Sources/WireIndividualToTeamMigrationUI/Views/Previews/TeamPlanSelectionPreview.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+import WireFoundation
+
+#Preview {
+    PageContainer(
+        content: {
+            TeamPlanSelectionView(
+                features: [
+                    .init(
+                        id: "console",
+                        description: .localizedMarkdown(
+                            key: "individualToTeam.planSelection.feature.adminConsole",
+                            bundle: .module
+                        )
```

Comment:
just a question: Was it a requirement to use markdown?
For static views I think I'd try to achieve the same without markdown.
I just briefly tried to separate feature name and description and I think we could achieve the same appearance.
We'd have to adjust our text styling though to return `Text`.
![Screenshot 2024-12-04 at 13 36 34](https://github.com/user-attachments/assets/2a8dd39f-8f08-4cc4-b606-54a229d35b57)

Reply:
How does the second text view in the stack wraps beneath the first one?